### PR TITLE
Cache user org role

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/orgs/tests.py [hunk 1]
```diff
@@ -3464,7 +3464,7 @@ def test_home(self):
 
         home_url = reverse("orgs.org_home")
 
-        with self.assertNumQueries(125):
+        with self.assertNumQueries(26):
```
Comment (rowanseymour) [inline temba/orgs/tests.py:3467]: hans-gruber-falling

File: temba/orgs/models.py [hunk 2]
```diff
@@ -1156,14 +1166,21 @@ def get_owner(self) -> User:
...
+        """
 
-        for role in OrgRole:
-            if self.get_users_with_role(role).filter(id=user.id).exists():
-                return role
+        def get_role():
+            if user.is_staff:
+                return OrgRole.ADMINISTRATOR
+            for role in OrgRole:
+                if self.get_users_with_role(role).filter(id=user.id).exists():
```
Comment (rowanseymour) [inline temba/orgs/models.py:1177]: i still want to improve upon this

File: temba/orgs/tests.py (hunk 1 above)
Comment (rowanseymour) [reply temba/orgs/tests.py:3467]: also note this was `159` before we added caching of role permissions

File: temba/orgs/models.py (hunk 2 above)
Comment (norkans7) [reply temba/orgs/models.py:1177]: How will the beta group be handled now?

Comment (rowanseymour) [reply temba/orgs/models.py:1177]: "Beta" isn't one of the groups we use for an org role so it's handled differently as just a cached property https://github.com/nyaruka/rapidpro/blob/8fd3489a48e1130360285ef95ee6c51b31b40623/temba/orgs/models.py#L237

**Review by norkans7 — APPROVED** [review]

**Pull request merged**